SSA/Core/Framework.lean

```diff
@@ -1201,17 +1201,34 @@ section SimpPeephole
 section SimpPeepholeApplier
 
 /-- rewrite with `pr` to `target` program, at location `ix` and later, running at most `fuel` steps. -/
-def rewritePeephole_go (fuel : ℕ) (pr : PeepholeRewrite Op Γ t) (ix : ℕ) (target : Com Op Γ₂ t₂) :
-    (Com Op Γ₂ t₂) := 
+def rewritePeephole_go (fuel : ℕ) (pr : PeepholeRewrite Op Γ t)
+    (ix : ℕ) (target : Com Op Γ₂ t₂) : (Com Op Γ₂ t₂) := 
   match fuel with
   | 0 => target
   | fuel' + 1 => 
      let target' := rewritePeepholeAt pr ix target
      rewritePeephole_go fuel' pr (ix + 1) target'
 
 /-- rewrite with `pr` to `target` program, running at most `fuel` steps. -/
-def rewritePeephole (fuel : ℕ) (pr : PeepholeRewrite Op Γ t) (target : Com Op Γ₂ t₂) :
-    (Com Op Γ₂ t₂) := rewritePeephole_go fuel pr 0 target
+def rewritePeephole (fuel : ℕ)
+    (pr : PeepholeRewrite Op Γ t) (target : Com Op Γ₂ t₂) : (Com Op Γ₂ t₂) := 
+  rewritePeephole_go fuel pr 0 target
```

Comment:
This seems to be a non-functional change, just formatting. Maybe mention this in the PR description.

Reply:
Updated the PR description.